tests/core/tools/manual/keystroke.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: feature, 4.9.0, 1338
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, basicstyles
+
+# TEST
```

Comment:
Redundant text - it's a test so there's no need to announce it.

It could include useful information e.g. what this test verifies, but putting "test" alone is not helpful.